tsl/src/remote/connection.c

```diff
@@ -2176,8 +2176,17 @@ remote_connection_begin_copy(TSConnection *conn, const char *copycmd, bool binar
 	if (binary && !send_binary_copy_header(conn, err))
 		goto err_end_copy;
 
+	if (PQsetnonblocking(pg_conn, 1))
```

Comment:
What's the actual return value here for success?

When reading this code, it seems like we are generating an error on success, but I guess it returns `false` on success? If we look at `PQisnonblocking()` below, it is the other way around. That looks like an inconsistency.

If the return value is `0` on success, I think we should explicitly check for that to make things clear, e.g., if `(PQsetnonblocking(pg_conn, 1) != 0)`